examples/multi_level.html

```diff
@@ -4,15 +4,16 @@
     <title>Multi Level Graph Example</title>
     <link rel="stylesheet" type="text/css" href="css/template.css" />
     <script src="../lib/ccNetViz.js"></script>
+    <script src="../lib//plugins/ccNetViz-multilevel-plugin.js"></script>
   </head>
 
   <body>
     <div class="canvas-container">
       <h3 class="title">Multi Level Graph</h3>
```

Comment:
It would be nice if you would be able to visualize the current level of the graph somewhere outside of the graph.

To have inside the div something like:
<>
"HELLO" -> "NESTED LEVEL" -> "EVEN NESTED MORE"
<>

Also these names can be clickable so there may be need for API for changing the levels (only going level up is neccessary).

Reply:
I have added this feature, also user can go to the parent graph only this way. Because onChangeViewport is very sensitive.